aiokafka/admin/client.py

```diff
@@ -97,13 +94,13 @@ def __init__(
         metadata_max_age_ms: int = 300000,
         security_protocol: str = "PLAINTEXT",
         ssl_context: SSLContext | None = None,
-        api_version: str = "auto",
         sasl_mechanism: str = "PLAIN",
         sasl_plain_username: str | None = None,
         sasl_plain_password: str | None = None,
         sasl_kerberos_service_name: str = "kafka",
         sasl_kerberos_domain_name: str | None = None,
         sasl_oauth_token_provider: str | None = None,
+        legacy_protocol: LegacyProtocol = False,
```

Comment:
```suggestion
        legacy_protocol: LegacyProtocol | None = None,
```

As it's `None` as default in downstream. 
Also default in doc string need to be updated.